server/views/pages/view/cases.njk

```diff
@@ -63,32 +63,51 @@
         }
     ]), licences) %}
 {% endfor %}
+
+{% if probationView %} {% set probationViewUrlQuery = '?view=probation' %} {% endif %}
+
 {% block content %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-three-quarters">
             <h1 class="govuk-heading-xl">Select a person to view or print licences and supervision orders</h1>
         </div>
     </div>
     <div class="moj-search govuk-!-margin-bottom-6">
-        <form method="GET">
-            <div class="govuk-form-group">
-                <label class="govuk-visually-hidden" for="search">
-                    Search for a case
-                </label>
-                <input class="govuk-input moj-search__input govuk-!-font-size-14" id="search" name="search" placeholder="You can search by name, prison number or probation practitioner" value="{{ search }}"></div>
-                <button class="govuk-button moj-search__button" data-module="govuk-button">
-                    Search
-                </button>
-            </form>
-        </div>
+        <nav class="moj-sub-navigation" aria-label="Sub navigation">
+            <ul class="moj-sub-navigation__list">
+                <li class="moj-sub-navigation__item">
+                    <a class="moj-sub-navigation__link" data-qa="prison-view-link"  {% if not probationView %}aria-current="page"{% endif %} href="?view=prison">People in prison</a>
```

Comment:
This perhaps links to the comment I put above, where we are passing `view=prison` as a query param, but if the `view` is present previously, we assume the link is to probation. It may be that these lines do not interact with each other in a way that causes an issue, but in my opinion it makes the code less clear.